dvc/command/machine.py

```diff
@@ -143,6 +143,21 @@ def run(self):
         return 0
 
 
+class CmdMachineStatus(CmdBase):
+    def run(self):
+        if self.repo.machine is None:
+            raise MachineDisabledError
+
+        all_status = self.repo.machine.status(self.args.name)
+        for i, status in enumerate(all_status):
```

Comment:
Use `start=1` instead of `i+1` below if you need 1-based indexing, makes intention clearer.
```suggestion
        for i, status in enumerate(all_status, start=1):
```